kitchen.yml

```diff
@@ -173,6 +173,19 @@ platforms:
     driver:
       image: netmanagers/salt-2017.7-py2:arch-base-latest
       run_command: /usr/lib/systemd/systemd
+<% if not ENV['CI'] == 'true' %>
+  - name: windows-81-2019-2-py3
+    driver:
+      name: vagrant
+      box: techneg/win81x64-pro-salt
+      linked_clone: true
+    provisioner:
+      init_environment: >
+        salt-call --local state.single file.managed
+        C:\Users\vagrant\AppData\Local\Temp\kitchen\srv\salt\win\repo-ng\openvpn.sls
+        source=https://github.com/saltstack/salt-winrepo-ng/raw/master/openvpn.sls
+        skip_verify=True makedirs=True
+<% end %>
```

Comment:
#### Proposal: Use `kitchen.vagrant.yml` for this part of the configuration

ci(kitchen): move Vagrant configuration to separate file

* https://kitchen.ci/docs/reference/configuration/
* Avoid `yamllint` errors
* Run with:

```bash
$ export KITCHEN_LOCAL_YAML=kitchen.vagrant.yml \
  && bin/kitchen verify ${INSTANCE}
```

---

With `kitchen.vagrant.yml` as:

```yaml
# -*- coding: utf-8 -*-
# vim: ft=yaml
---
driver:
  name: vagrant

platforms:
  - name: windows-81-2019-2-py3
    driver:
      box: techneg/win81x64-pro-salt
      customize:
        usbxhci: 'off'
      gui: false
      linked_clone: true
    provisioner:
      init_environment: >
        salt-call --local state.single file.managed
        C:\Users\vagrant\AppData\Local\Temp\kitchen\srv\salt\win\repo-ng\openvpn.sls
        source=https://github.com/saltstack/salt-winrepo-ng/raw/master/openvpn.sls
        skip_verify=True makedirs=True
```

Reply:
Ok, I can see the advantages of this. I'd forgotten about `KITCHEN_LOCAL_YAML` and didn't won't to duplicate the whole file.